src/components/QuranReader/TranslationView/TranslationViewCell.module.scss

```diff
@@ -16,6 +16,24 @@
   padding-top: var(--spacing-medium);
   position: relative;
 }
+
+.splitView {
```

Comment:
It looks like the page breaks if no translations are selected. Not sure if it's a previous bug